db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20210407170804) do
+ActiveRecord::Schema.define(version: 20210414171109) do
```

Comment:
We'll have to change this to `structure.sql` at some point. It's not the first time we see that `schema.rb` doesn't support what we do in a migration hence, doing a schema load locally won't mimic production's schema 🙈

Reply:
What does that change look like? We'd be updating this line to be a different class? 

Are there other things besides constraints that don't get reflected as a result? Rails 6.1 has support for constraints in migrations, but I don't know that it would also change the schema.rb file.

Reply:
[This](https://github.com/coopdevs/timeoverflow/commit/e09469ce1aae16b0728fdeb3c3f86ecbdd61217d#diff-1dd5a8f580b9615769ddc19a64a98f455d9d20eb149c443f41d9d3c797d39b12) is how it looks. Essentially, while `schema.rb` is DBMS-agnostic, `structure.sql` is not and therefore, it can handle anything our DB supports. https://blog.appsignal.com/2020/01/15/the-pros-and-cons-of-using-structure-sql-in-your-ruby-on-rails-application.html covers the topic quite well.